### ISSUE TITLE
Init Error Haven't Used Cloud Storage

### ISSUE THREAD

**Issue by sanjanaitang** [firebase/firebase-tools#2316]
### [REQUIRED] Environment info

**firebase-tools:** 8.4.1

**Platform:** Ubuntu

### [REQUIRED] Steps to reproduce

```
firebase init
Which Firebase CLI features do you want to set up for this folder? Press Space to select features, then Enter to confirm your choi
ces. Firestore: Deploy rules and create indexes for Firestore, Functions: Configure and deploy Cloud Functions, Hosting: Configure a
nd deploy Firebase Hosting sites, Storage: Deploy Cloud Storage security rules
Please select an option: Use an existing project
```
...and follow the next steps as needed.
My existing project uses Cloud Storage and has a Storage bucket that works as it should.

### [REQUIRED] Expected behavior

Ended without complaint related to the use of Cloud Storage.

### [REQUIRED] Actual behavior

`Error: It looks like you haven't used Cloud Storage in this project before. Go to https://console.firebase.google.com/project/<my-existing-project>/storage to create your Storage bucket.`

**Comment by samtstern** [comment]
@sanjanaitang that error appears when the Cloud Storage API is not enabled.  Have you followed the link in the error message?  If so, do you see a Cloud Storage bucket?  Also is there any information in `firebase-debug.log`?

**Comment by sanjanaitang** [comment]
```
Have you followed the link in the error message?
Yes I have.
```

```
If so, do you see a Cloud Storage bucket?
Yes I do.
```

```
Also is there any information in firebase-debug.log?
=== Storage Setup
[debug] [2020-06-01T12:22:26.471Z] >>> HTTP REQUEST GET https://serviceusage.googleapis.com/v1/projects/my-project/services/firebasestorage.googleapis.com  
 
[debug] [2020-06-01T12:22:27.882Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Mon, 01 Jun 2020 12:22:27 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[error] 
[error] Error: It looks like you haven't used Cloud Storage in this project before. Go to https://console.firebase.google.com/project/my-project/storage to create your Storage bucket.
```
![Screenshot from 2020-06-01 19-49-44](https://user-images.githubusercontent.com/66248819/83411133-d749f800-a441-11ea-8c69-67c6e975498e.png)

**Comment by samtstern** [comment]
@sanjanaitang thank you.  I was able to confirm that this is a bug and reproduce it myself.  Fix on the way.  

In the meantime you will want to use `firebase-tools` version `8.4.0`

**Comment by sanjanaitang** [comment]
@samtstern thank you.

**Comment by Barthold-Albrecht** [comment]
I get the same error (with version 8.4.1). Is this supposed to be fixed already?

**Comment by samtstern** [comment]
@Barthold-Albrecht the fix has been merged and not released, it will be included in the next version (which will be either 8.4.2 or 8.5.0)

**Comment by Barthold-Albrecht** [comment]
Thank you for the info @samtstern ! When will the new version be released? And if this still takes a while: should I downgrade to 8.4.0 as well?

**Comment by samtstern** [comment]
I can't really say when, normally we release every week or two.  And yes you should downgrade for now, sorry about the inconvenience.

**Comment by chirmac** [comment]
Also got the same error, have a bucket configured but the CLI doesn't recognize it. Exact same logs and results. Will downgrade.

**Comment by Barthold-Albrecht** [comment]
> Also got the same error, have a bucket configured but the CLI doesn't recognize it. Exact same logs and results. Will downgrade.

It worked for me!

**Comment by chirmac** [comment]
Just downgraded, works as intended now.

> > Also got the same error, have a bucket configured but the CLI doesn't recognize it. Exact same logs and results. Will downgrade.
> 
> It worked for me!

**Comment by samtstern** [comment]
Hey everyone this is now fixed in version `8.4.2` ... please upgrade!
